pages/your_domain/single_cell.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Single-Cell Sequencing"
+description: "Managing data generated from single-cell sequencing experiments."
+contributors: [Johan Rollin, Pavankumar Videm, MeInBIo program, Freiburg University]
```

Comment:
```suggestion
contributors: [Johan Rollin, Pavankumar Videm]
```

Only people are listed here. If you want to specify the affiliation, please do so by adding your name + information to the `/_data/CONTRIBUTORS.yml` file!